From: Misaka Build Factory
Subject: Fix GCC 15 missing sys/select.h include

GCC 15 no longer implicitly includes sys/select.h from other headers.

--- a/libavformat/tls_gnutls.c
+++ b/libavformat/tls_gnutls.c
@@ -24,6 +24,7 @@
 #include <gnutls/gnutls.h>
 #include <gnutls/x509.h>
 
+#include <sys/select.h>
 #include "avformat.h"
 #include "internal.h"
 #include "network.h"
